src/patterns/organisms/footer/includes/footer-section-company.hbs

```diff
@@ -0,0 +1,10 @@
+<section class="mzp-c-footer-section">
+  <h5 class="mzp-c-footer-heading">
+    Company
+  </h5>
+  <ul class="mzp-c-footer-list">
+    <li><a href="https://www.mozilla.org/en-US/about/">About</a></li>
```

Comment:
Nit: remove the hardcoded locales from example links. We would know to remove them in bedrock, but others teams might cut and paste from the examples here.